sdks/java/core/src/main/java/org/apache/beam/sdk/util/BigQueryTableInserter.java

```diff
@@ -95,6 +94,22 @@ public BigQueryTableInserter(Bigquery client) {
     this.client = client;
```

Comment:
If we do that, we gonna be breaking compatibility with previous versions. Is that ok?